lib/retriable.rb

```diff
@@ -36,8 +36,11 @@ def retriable(opts = {})
     on                = local_config.on
     on_retry          = local_config.on_retry
     sleep_disabled    = local_config.sleep_disabled
+    ignore            = local_config.ignore
 
-    exception_list = on.is_a?(Hash) ? on.keys : on
+    exception_list = on.is_a?(Hash) ? on.keys : [on].flatten
```

Comment:
Why are you wrapping it in an array and flattening? Just in case on is a single Exception?

Reply:
That's right yes. I was just making sure that this always returns an array, even if we pass a single object in the params.